src/base_model.jl

```diff
@@ -103,26 +113,26 @@ function build_base_model!(ECModel::AbstractEC, optimizer; use_notations=false)
     # Design of assets of the user
     @variable(model_user,
         0 <= x_us[u=user_set, a=device_names(users_data[u])]
-            <= field_component(users_data[u], a, "max_capacity"))
+            <= field_component(users_data[u], a, "max_capacity")/field_component(users_data[u], a, "nom_capacity"), Int)

```

Comment:
I wouldn't force the integer format; for this PR, I'd keep it continuous unless necessary or integer by option, such as in the case of the therm that there is the option "integer"